scripts/lint_prowjobs.go

```diff
@@ -0,0 +1,279 @@
+/*
```

Comment:
nit: can you rename this file `main.go` and place it in `/scripts/lint_prowjobs/main.go`?